pkgs/build-support/agda/default.nix

```diff
@@ -73,6 +76,9 @@ let
           find \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
           runHook postInstall
         '';
+
+        # Retrieve the current package from the finished package set and try to build it
+        passthru.tests.build = builtins.getAttr pname agdaPackages;
```

Comment:
@alexarice How about this? If it works I can drop the other commits and leave the nixos test as it is.

Reply:
Come to think about it, I find this strange. All this does is to check whether the package builds. That should be covered by saying "ofborg build agdaPackages.mylibrary" shouldn't it?